scanpipe/templates/scanpipe/panels/file_tree_panel.html

```diff
@@ -0,0 +1,29 @@
+<ul class="pl-2">
+  {% for node in children %}
+    <li class="mb-1">
+      {% if node.is_dir %}
+        <div class="tree-node is-flex is-align-items-center has-text-weight-semibold is-clickable px-1" data-folder{% if node.has_children %} data-target="{{ node.path|slugify }}" data-url="{% url 'file_tree' slug=project.slug %}?path={{ node.path }}"{% endif %}>
+          <span class="icon is-small chevron mr-1" data-chevron{% if not node.has_children %} style="visibility: hidden;"{% endif %}>
+            <i class="fas fa-chevron-right"></i>
+          </span>
+          <span class="is-flex is-align-items-center folder-meta">
+            <span class="icon is-small mr-1">
+              <i class="fas fa-folder"></i>
+            </span>
+            <span>{{ node.name }}</span>
+          </span>
+        </div>
+        {% if node.has_children %}
+          <div id="dir-{{ node.path|slugify }}" class="ml-4 hidden" data-loaded="false"></div>
+        {% endif %}
+      {% else %}
+        <div class="is-flex is-align-items-center ml-5 is-clickable">
+          <span class="icon is-small mr-1">
+            <i class="fas fa-file"></i>
```

Comment:
Let's use `far` to enforce the visual distinction between files and directories.